semana-1/portfolio/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tudo Sobre Eu</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="xuxu">
+        <a class="xuxu" href="http://127.0.0.1:5500/Projeto.html"> Projetos </a>
```

Comment:
Aqui voce deveria indicar apenas o path/caminho para o arquivo Projeto.html. Quando for dar nome pra variáveis e nomes de classe, tente colocar algo que faca sentido pras pessoas que forem ler seu codigo e que realmente expliquem o que é feito ali 

`` 
```suggestion
        <a class="xuxu" href="./Projeto.html"> Projetos </a>
```

Reply:
eu tinha feito pra testar e esqueci de alterar. Desculpa.

=-//